### PR TITLE
Update DAKE target

### PR DESCRIPTION
**Pull request by ThanYangFPV**
Hello everyone.

I'm adding new sensors for DAKEFPVF405 and DAKEFPVF722, please check. 

Added HMC5883 and QMC5883 for DAKEFPVF405. 
Added HMC5883 and QMC5883 and ICM42688 for DAKEFPVF722.

Thank you all.

### CODE REVIEW

File: configs/default/DAKE-DAKEFPVF405.config [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #define USE_BARO
 #define USE_BARO_SPI_BMP280
 #define USE_BARO_SPI_DPS310
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```
Comment (haslinghuis) [inline configs/default/DAKE-DAKEFPVF405.config:16]: Does the flight controller include the magnetometer on board ?

Comment (ThanYangFPV) [reply configs/default/DAKE-DAKEFPVF405.config:16]: > Does the flight controller include the magnetometer on board ?

No, the board has SDA and SCL pads for connecting the compass.

Comment (haslinghuis) [reply configs/default/DAKE-DAKEFPVF405.config:16]: Okay in that case they need to be removed. They will be included when user adds magnetometer option during cloud build.

Comment (haslinghuis) [inline configs/default/DAKE-DAKEFPVF405.config:16]:
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/DAKE-DAKEFPVF722.config [hunk 2]
```diff
@@ -3,12 +3,16 @@
...
+#define USE_GYRO_SPI_ICM42688P
 #define USE_ACC
 #define USE_ACC_SPI_MPU6000
 #define USE_ACC_SPI_MPU6500
+#define USE_ACC_SPI_ICM42688P
 #define USE_BARO
 #define USE_BARO_SPI_BMP280
 #define USE_BARO_SPI_DPS310
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```
Comment (haslinghuis) [inline configs/default/DAKE-DAKEFPVF722.config:15]:
```suggestion
```

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**